nixos/modules/services/video/frigate.nix

```diff
@@ -114,6 +132,20 @@ in
       '';
     };
 
+    vaapiDriver = mkOption {
+      type = nullOr (enum [ "i965" "iHD" "nouveau" "vdpau" "nvidia" "radeonsi" ]);
+      default = null;
+      example = "radeonsi";
+      description = ''
+        VA-API driver to use for video acceleration. Use together with `settings.ffmpeg.hwaccel_args`.
+
+        - https://docs.frigate.video/configuration/hardware_acceleration
+        - https://docs.frigate.video/configuration/ffmpeg_presets#hwaccel-presets
+
+        Requires a VA-API driver to be installed through `hardware.graphics.extraPackages`.
```

Comment:
```suggestion
        Force usage of a particular VA-API driver for video acceleration. Use together with `settings.ffmpeg.hwaccel_args`.

        Setting this is not required for VA-API to work but it can help steer it towards the correct card if you have multiple.

        :::{.note}
        You must configure a VA-API driver package for VA-API to work, regardless of this setting.
        AMDGPUs require {option}`hardware.graphics.enable` and most Intel GPUs need `pkgs.intel-media-driver` in {option}`hardware.graphics.extraPackages`.
        :::

        See also:

        - https://docs.frigate.video/configuration/hardware_acceleration
        - https://docs.frigate.video/configuration/ffmpeg_presets#hwaccel-presets
```

Reply:
Again, are option descriptions rendered with admonitions *anywhere at all*?

Reply:
It renders fine in the options manual:

    nix-build nixos/release.nix -A manual.x86_64-linux && xdg-open ./result/share/doc/nixos/options.html

Reply:
I also just confirmed that it renders fine in `man configuration.nix` too.

s.n.o currently just strips the admonition text but keeps the content as if it was regular content. See i.e. https://search.nixos.org/options?channel=unstable&show=security.dhparams.params&from=0&size=50&sort=relevance&type=packages&query=security.dhparams.params where the last 3 paragraphs are note, warning and note.

Once it gets proper support, it should simply "just work".